hardhat/scripts/clients.ts

```diff
@@ -0,0 +1,31 @@
+import hre from 'hardhat';
```

Comment:
is this file just to try getting basics working?

Reply:
add comment that this is example of using viem instead of ethers with hardhat

Reply:
done.